src/DIRAC/Core/Tornado/scripts/tornado_start_all.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+########################################################################
+# File :   tornado-start-all
+# Author : Louis MARTIN
+########################################################################
+# Just run this script to start Tornado and all services
+# Use CS to change port
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+__RCSID__ = "$Id$"
+
+# Must be define BEFORE any dirac import
+import os
+import sys
+from DIRAC.Core.Utilities.DIRACScript import DIRACScript
+
+
+@DIRACScript()
+def main():
+  os.environ['DIRAC_USE_TORNADO_IOLOOP'] = "True"
```

Comment:
what's wrong with keeping it at the top ?

Reply:
Discussed in https://github.com/DIRACGrid/DIRAC/pull/4865#discussion_r557381856